GoodMorning/GoodMorning/OnBoarding/OnBoardingViewController.swift

```diff
@@ -0,0 +1,216 @@
+            subView: self.birthDateTextField
+        )
+
+        return birthDateStackView
+    }()
+
+    private lazy var birthDateTextField: UITextField = {
+        let birthDateTextField = UITextField()
+        birthDateTextField.font = .pretendard(size: 20, weight: .bold)
+        birthDateTextField.layer.cornerRadius = 10
+        birthDateTextField.backgroundColor = .lightGray.withAlphaComponent(0.3)
+        birthDateTextField.inputView = self.birthDatePicker
+        birthDateTextField.placeholder = "생년월일을 작성해주세요."
+        birthDateTextField.textAlignment = .center
+
+        return birthDateTextField
+    }()
+
+    private lazy var birthDatePicker: UIDatePicker = {
+        let birthDatePicker = UIDatePicker()
+        birthDatePicker.datePickerMode = .date
+        birthDatePicker.preferredDatePickerStyle = .wheels
+        birthDatePicker.addTarget(
+            self,
+            action: #selector(datePickerValueDidChanged),
+            for: .valueChanged
+        )
+
+        return birthDatePicker
+    }()
+
+    private let genderStackView: UserInformationStackView = {
+        let font = UIFont.pretendard(size: 20, weight: .medium)
+        let segmentedControl = UISegmentedControl(items: ["여자", "남자"])
+        segmentedControl.selectedSegmentIndex = 0
+        segmentedControl.setTitleTextAttributes([NSAttributedString.Key.font: font], for: .normal)
+
+        let genderStackView = UserInformationStackView(title: "성별", subView: segmentedControl)
+
+        return genderStackView
+    }()
+
+    private let startGoodMorningButton: UIButton = {
+        let startGoodMorningButton = UIButton()
+        startGoodMorningButton.layer.cornerRadius = 15
+        startGoodMorningButton.setTitle("굿모닝 시작하기", for: .normal)
+        startGoodMorningButton.setTitleColor(.black, for: .normal)
+        startGoodMorningButton.titleLabel?.font = .pretendard(size: 30, weight: .medium)
+        startGoodMorningButton.backgroundColor = .design(.mainBackground)
+
+        return startGoodMorningButton
+    }()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.view.backgroundColor = .white
+        configureSubViews()
+        addKeyBoardObserver()
+        setUpDatePickerToolBar()
+    }
+
+    private func configureSubViews() {
+        self.view.addSubview(self.greetingStackView)
+        self.view.addSubview(self.userInputStackView)
+        self.view.addSubview(self.startGoodMorningButton)
+        configureConstraints()
+    }
+
+    private func setUpDatePickerToolBar() {
+        let space = UIBarButtonItem(barButtonSystemItem: .flexibleSpace, target: nil, action: nil)
+        let doneButton = UIBarButtonItem(
+            barButtonSystemItem: .done,
+            target: self,
+            action: #selector(doneButtonDidTapped)
+        )
+        let toolBar = UIToolbar(frame: CGRect(x: 0, y: 0, width: 100, height: 100))
+        toolBar.items = [space, doneButton]
+        toolBar.sizeToFit()
```

Comment:
textField.inputAccessoryView를 넣었을 때 autolayout 에러가 발생하는 문제에 대해서 저도 찾아보았어요!
물론 toolBar의 frame 값을 넣으면 해결되는 문제이긴 하지만, 사실 이 값은 toolBar의 실제 frame 값이 아니기 때문에 의미 없는 코드가 아닐까 생각했어요.
여러 글을 참고하고 실험해 보니,

```suggestion
        let toolBar = UIToolbar()
        toolBar.sizeToFit()
        toolBar.items = [space, doneButton]
```
toolbar의 사이즈를 먼저 잡아주고, item을 추가하면 auto layout 관련 에러가 나지 않아요!

추가로 아주 사소한 것인데,, 혹시 해당 속성의 이름을 `toolbar`로 변경하는 것은 어떤가요? UIKit에서는 보통 ~bar를 소문자로 쓰더라구요😀

Reply:
앗 써니! 저는 순서를 바꿔도 계속 레이아웃 오류가 나오네요..🥲 무슨 문제일까요?!..
그래도 써니 시뮬레이터에서는 문제가 없다고 하니 말씀해주신대로 순서를 바꾸고 toolbar로 네이밍도 바꿔보았습니다!

> 해당커밋: c457583

Reply:
?!!!!! 정말요>?!!!!! 저 제 핸드폰으로 연결해서 해봐도 오류가 안나는데 이유가 무엇일까요...?

Reply:
혹시 써니 그 Textfield 터치시 keyboard가 팝업되고 없어지는 거는 잘 되나요?? 이상하게 코드로는 문제가 없는데, 시뮬레이터에서는 한번씩 UI 오류가 나네요..